lagotto/etc/systemd/system/lagotto.service

```diff
@@ -0,0 +1,18 @@
+{% from "lagotto/map.jinja" import props with context -%}
+{% set alm_root = props.get('app_root') -%}
+[Unit]
+Description=Lagotto
+Requires=network.target
+
+[Service]
+Type=simple
+Environment="HOME=/home/lagotto"
+Environment="ALM_ROOT={{ alm_root }}"
+User=lagotto
+Group=lagotto
+ConditionPathIsDirectory={{ alm_root }}/current
+WorkingDirectory={{ alm_root }}/current
+ExecStart=/bin/bash -lc 'chruby-exec 2.2.2 -- bundle exec puma -i 0 -C $ALM_ROOT/shared/puma.rb'
```

Comment:
The "-i 0" arg refers to instance #0 in the upstart script. the intent was to only run one instance of this service. Not sure how to do this in systemd.

Reply:
The Ruby version is in the property map ("version_ruby"). Maybe can refer to that here. We will be upgrading Ruby eventually.

Reply:
I'll put the `version_ruby` in here, that's a solid idea. There are other ways to track running instances, I'd expect that this would still work, but I'm happy to try a more "standard" approach like tracking the PID.

Reply:
Everything seems ok with the `-i 0` removed, doing a bit more testing but it seems ok